src/components/Breadcrumbs/index.tsx

```diff
@@ -0,0 +1,75 @@
+import Link from 'next/link';
+import { Breadcrumbs } from '@aws-amplify/ui-react';
+import { findDirectoryNode as findNode } from '@/utils/findDirectoryNode';
+
+type BreadcrumbItem = {
+  href: {pathname:string, query?: { platform: string }};
+  label: string;
+  isDisabled?: boolean;
+};
+
+type Props = {
+  route: string;
+  platform: string;
+};
+
+const overrides = {
```

Comment:
Not sure what kind of usage this will get but a basic override system, currently used to turn '/' to 'Home' but in the future might be used for turning things like '/android/prev' to 'V0' while also changing '/javascript/prev' to 'V6(preview)'  Anyway we can remove this if you all want and just put in a straight replace '/' for 'Home' and nothing else